components/util/geometry.rs

```diff
@@ -84,6 +84,15 @@ impl Zero for Au {
     }
 }
 
+impl fmt::Display for Au {
+    fn fmt(&self, formatter: &mut Formatter) -> fmt::Result {
+        let Au(app_unit) = self.clone();
+        let tmp = app_unit.to_string();
+        try!(formatter.write_str(tmp.as_slice()));
+        Ok(())
+    }
+}
```

Comment:
@bjz when I first tried to implement this, the compiler barked that there was no way to print a Au and it was missing the Show/String ops.